### PR TITLE
user is added

### PR DESCRIPTION
**Pull request by kiranwannerkar**

### CODE REVIEW

File: src/main/java/com/ty/springmvc_dummy/dto/User.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.ty.springmvc_dummy.dto;
+
+public class User {
+private int id;
+private String name;
+private long phone;
```
Comment (mansoorad) [inline src/main/java/com/ty/springmvc_dummy/dto/User.java:6]: add one more attribute address

File: src/main/java/com/ty/springmvc_dummy/dto/User.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+public void setName(String name) {
+	this.name = name;
+}
+public long getPhone() {
+	return phone;
+}
+public void setPhone(long phone) {
+	this.phone = phone;
+}
```
Comment (darshandk739) [inline src/main/java/com/ty/springmvc_dummy/dto/User.java:31]: 1.Override toString() now
2.Add some more attributes

**Review by darshandk739 — COMMENTED** [review]
good code

**Review by darshandk739 — APPROVED** [review]
Good

**Review by mansoorad — APPROVED** [review]
review done

**Pull request merged**